<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotAsymmetryByLogValueDialog</class>
 <widget class="QDialog" name="PlotAsymmetryByLogValueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotAsymmetryByLogValue</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="runsLayout">
        <item>
         <widget class="QLabel" name="runsLabel">
          <property name="text">
           <string>Runs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MantidQt::API::FileFinderWidget" name="runs" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="label" stdset="0">
           <string/>
          </property>
          <property name="multipleFiles" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Log value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="logBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_41">
          <item>
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Function</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="typeBoxLog">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="OutputWSLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Output workspace</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outWSBox"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="typeBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Periods</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="redBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="greenBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Grouping</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Forward spectra</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="forwardBox">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Backward spectra</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="backwardBox"/>
        </item>
        <item>
         <widget class="QLabel" name="alphaLabel">
          <property name="text">
           <string>Alpha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="alpha">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeMinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeMaxBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dtc">
     <property name="title">
      <string>Dead Time Correction</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="dtcTypeLayout" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="dtcTypeLabel">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dtcType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="dtcFileContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="dtcFileLabel">
           <property name="text">
            <string>File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dtcFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="dtcFileBrowseButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::API::FileFinderWidget</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/FileFinderWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>logBox</tabstop>
  <tabstop>outWSBox</tabstop>
  <tabstop>redBox</tabstop>
  <tabstop>forwardBox</tabstop>
  <tabstop>timeMinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
